src/index.html

```diff
@@ -11,8 +11,94 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,300&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Consider combining the 'preconnect' links and the font stylesheet into a single link tag to reduce the number of HTTP requests. You can use the 'display=swap' parameter in the URL to ensure text remains visible during webfont load.